src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -341,6 +341,7 @@ def feature_enabled?(feature)
   def feature_css_class
     css_classes = []
     css_classes << 'responsive-ux' if feature_enabled?(:responsive_ux)
+    css_classes << 'user-profile-redesign' if feature_enabled?(:user_profile_redesign)
```

Comment:
If this new CSS class is not used, I would not introduce it. Is it needed?

Reply:
I added this as a separate commit, as this is not directly related to the feature, but it should be fixed. I added this after discussion with Saray. I can move it to another PR, but I thought that this one line may be added here as a separate commit.

Reply:
In order to avoid any confusion, I've removed the commit from this PR and will introduce the change in another PR.